src/lib/cmd.rs

```diff
@@ -7,16 +7,27 @@ fn remote_cmd(args: &mut Vec<&str>, output: &mut File) -> Result<(), String> {
     // TODO(azhng): let's finger cross this works.
     let output_file = output.try_clone().unwrap();
     let error_file = output.try_clone().unwrap();
-    let cmd = Command::new("ssh")
+    let cmd_process = Command::new("ssh")
         .arg(config::SEXXI_REMOTE_HOST)
-        .args(args)
+        .args(args.clone())
```

Comment:
Hmm can we only pass in args as a reference here instead of cloning?

Reply:
I have to clone the args. Not sure how else to do it.

Reply:
Like i'm assuming it was working before u called clone?

Reply:
You can just force it to be passed as a reference instead of using move semantics

``` rust
let cmd_process = Command::new("ssh")
       .args(&*args)
```

Reply:
That is move semantics still, why do we need to move?

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=ff1b2646aaa11194997fd89a3a0d8df8

Reply:
I don't quite understand the code snippet here. Why are you trying to assign a `String` to a `&String` ?